src/main/java/org/pesmypetcare/webservice/service/appmanager/GoogleCalendarServiceImpl.java

```diff
@@ -29,6 +29,9 @@
     @Override
     public void createSecondaryCalendar(String accessToken, String owner, String petName)
         throws CalendarAccessException, DatabaseAccessException, DocumentException {
+        if (petDao.getSimpleField(owner, petName, CALENDAR_ID) != null) {
+            throw new CalendarAccessException("409", "This pet already has a secondary calendar");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "409" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)